### DISCUSSION TITLE
I want to change the background of Page. I have HTML and CSS, how can I integrate?

### DISCUSSION THREAD

**Discussion by anubhavrohatgi** [writer/writer-framework#254] [Q&A]
I want to change the background of Page. I have HTML and CSS, how can I integrate them? This will form the default theme of my project.

Ref: https://codepen.io/sarazond/pen/LYGbwj

**Comment by FabienArcellier** [comment]
Hi @anubhavrohatgi, you will have to think outside the box. Streamsync does not have an official mechanism for injecting structure into the page to build this sort of background.

I offer you a css + javascript hack. Be aware, thinking outside the box exposes you to seeing this mechanism stop working when streamsync is updated.

![Peek 2024-02-26 07-34](https://github.com/streamsync-cloud/streamsync/assets/159559/1965fb8f-97eb-4120-ad17-7c3475d95cf7)

This commit details the changes to be made: https://github.com/FabienArcellier/spike-streamsync-complex-background/commit/b23625709d546c6f9659538f3b0f4ed984ed48f5.

**Reply by anubhavrohatgi** [reply]
Hi @FabienArcellier,

Thanks for your prompt support. You have certainly encouraged me to look forward to this promising tech. Streamsync is a beautiful one-stop solution to all my front-end woes.

**Reply by anubhavrohatgi** [reply]
[Screencast from 26-02-24 10:17:59 PM IST.webm](https://github.com/streamsync-cloud/streamsync/assets/2736416/cb00c530-6157-4c0d-8245-cdf26f404dc7)

Check this the page height is automatically increasing or the background is moving without using the scroll
